.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,48 @@
+name: Feature request
+description: Suggest an idea for this project
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this issue! :hugs:
+        
+        However note the following before opening an issue:
+        * IF YOU DON'T FILL IN THE TEMPLATE PROPERLY, YOUR ISSUE IS LIABLE TO BE CLOSED.
```

Comment:
People who write in caps won't be taken seriously. The same applies here. If you want to emphasize some of this, you can format it in bold letters. However, it's redundant to the checkboxes.

Reply:
After line 12 maybe add something similar to the following?

    * Requests for any new service such as WaybackMachine, TikTok, VipTube, etc. need to be suggested at [/NewPipeExtractor search](https://github.com/TeamNewPipe/NewPipeExtractor/issues?q=+is%3Aissue+label%3A%22new+service%22 )  first before submitting.